public/gameInfo.php

```diff
@@ -57,7 +53,7 @@
 $richPresenceData = $gameData['RichPresencePatch'];
 
 // Entries that aren't actual game only have alternatives exposed, e.g. hubs.
-$isFullyFeaturedGame = !in_array($consoleName, ['Hubs']);
+$isFullyFeaturedGame = $consoleName !== 'Hubs';
 
```

Comment:
Case sensitive? Maybe not worth writing about

Reply:
hrm, it assumes the name of this "console" never changes. true. guess it would be safer to compare the consoleId instead